Svc/DpWriter/DpWriter.hpp

```diff
@@ -32,6 +32,11 @@ class DpWriter : public DpWriterComponentBase {
     //!
     ~DpWriter();
 
+    //! configure writer
+    void configure(
+            const Fw::String& dir //!< directory for file writing
```

Comment:
Maybe this should be Fw::FileNameString?

Reply:
Changed them to `Fw::FileNameString`

Reply:
Looks good. In my edits I changed the type in the configure prototype to `const Fw::StringBase&`. I think we should use this pattern going forward whenever the concrete string type is not needed.